docs/hdfs_connector.rst

```diff
@@ -54,16 +82,37 @@ The three records entered are published to the Kafka topic ``test_hdfs`` in Avro
 
 Before starting the connector, please make sure that the configurations in
 ``etc/kafka-connect-hdfs/quickstart-hdfs.properties`` are properly set to your configurations of
-Hadoop, e.g. ``hdfs.url`` points to the proper HDFS and using FQDN in the host. Then run the
-following command to start Kafka connect with the HDFS connector::
+Hadoop, e.g. ``hdfs.url`` points to the proper HDFS and using FQDN in the host. Then start connector by loading its
+configuration with the following command:
+
+.. sourcecode:: bash
+
+   $ confluent load hdfs-sink -d etc/kafka-connect-hdfs/quickstart-hdfs.properties
+   {
```

Comment:
To keep uniformity with other quickstarts in other repos, I'm inclined to leave it as is for now, and issue a separate PR. This seems to be a change will small impact. 

I agree with the observation. At the same time someone could argue that all the readers that are ready to use Avro format to ingest their data in our quickstarts won't have a very hard time with this one. Let's reconsider separately soon after the quickstarts are updated.